.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -33,14 +35,15 @@ body:
       description: Please paste any relevant log output or stacktrace if you're getting an exception/crash.
       render: shell
 # Repro information
-  - type: dropdown
+ - type: dropdown
     id: repro
     attributes:
       label: Can you reproduce the bug?
       options:
-        - Yes, always
-        - Yes, sometimes
-        - Not yet
+        - -- select --
+        - Always
+        - Sometimes
+        - 'No'
```

Comment:
```suggestion
        - No
```

Reply:
This is intentional. yaml don't deal well with a "boolean" type and strings...